# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../asterisk-sounds/sounds_cleanup_dups.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- asterisk-sounds-1.2.1/Makefile.orig	2006-04-02 19:27:58.000000000 +0200
+++ asterisk-sounds-1.2.1/Makefile	2006-04-02 19:29:12.000000000 +0200
@@ -59,19 +59,10 @@
 
 datafiles: all
 	mkdir -p $(DESTDIR)$(ASTVARLIBDIR)/sounds
-	mkdir -p $(DESTDIR)$(ASTVARLIBDIR)/sounds/letters
 	mkdir -p $(DESTDIR)$(ASTVARLIBDIR)/sounds/phonetic
 	mkdir -p $(DESTDIR)$(ASTVARLIBDIR)/sounds/silence
 	mkdir -p $(DESTDIR)$(ASTVARLIBDIR)/sounds/wx
 	mkdir -p $(DESTDIR)$(ASTVARLIBDIR)/sounds/ha
-	for x in sounds/letters/*.gsm; do \
-		if grep -q "^%`basename $$x`%" sounds-extra.txt; then \
-			install -m 644 $$x $(DESTDIR)$(ASTVARLIBDIR)/sounds/letters ; \
-		else \
-			echo "No description for $$x"; \
-			exit 1; \
-		fi; \
-	done
 	for x in sounds/phonetic/*.gsm; do \
 		if grep -q "^%`basename $$x`%" sounds-extra.txt; then \
 			install -m 644 $$x $(DESTDIR)$(ASTVARLIBDIR)/sounds/phonetic ; \
